src/pencilkit.cs

```diff
@@ -0,0 +1,243 @@
+//
+// PencilKit C# bindings
+//
+// Authors:
+//	TJ Lambert  <t-anlamb@microsoft.com>
+//
+// Copyright 2019 Microsoft Corporation All rights reserved.
+//
+
+#if MONOMAC
+using AppKit;
+using UIColor = AppKit.NSColor;
+using UIImage = AppKit.NSImage;
+
+using UIScrollViewDelegate = Foundation.NSObjectProtocol;
+using UIScrollView = Foundation.NSObject;
+using UIGestureRecognizer = Foundation.NSObject;
+using UIResponder = Foundation.NSObject;
+using UIView = Foundation.NSObject;
+using UIWindow = Foundation.NSObject;
+#else
+using UIKit;
+#endif
+
+using System;
+using ObjCRuntime;
+using Foundation;
+using CoreGraphics;
+
+namespace PencilKit {
+
+	[iOS (13, 0), NoMac]
+	[Native]
+	enum PKEraserType : long {
+		Vector,
+		Bitmap,
+	}
+
+	[iOS (13, 0), NoMac]
+	enum PKInkType {
+		[Field ("PKInkTypePen")]
+		Pen,
+
+		[Field ("PKInkTypePencil")]
+		Pencil,
+
+		[Field ("PKInkTypeMarker")]
+		Marker,
+	}
+
+	[iOS (13, 0), NoMac]
+	[BaseType (typeof (NSObject))]
+	[Model (AutoGeneratedName = true)] [Protocol]
+	interface PKCanvasViewDelegate : UIScrollViewDelegate {
+
+		[Export ("canvasViewDrawingDidChange:")]
+		void DrawingDidChange (PKCanvasView canvasView);
+
+		[Export ("canvasViewDidFinishRendering:")]
+		void DidFinishRendering (PKCanvasView canvasView);
+
+		[Export ("canvasViewDidBeginUsingTool:")]
+		void DidBeginUsingTool (PKCanvasView canvasView);
+
+		[Export ("canvasViewDidEndUsingTool:")]
+		void EndUsingTool (PKCanvasView canvasView);
+	}
+
+	interface IPKCanvasViewDelegate {}
+
+	[iOS (13, 0), NoMac]
+	[BaseType (typeof (UIScrollView))]
+	interface PKCanvasView : PKToolPickerObserver {
+
+		// This exists in the base class
```

Comment:
Should this comment be left in?

Reply:
not strictly needed - but it explains why it's missing (which someone could comment on)